distributions/aws/examples/rds-s3/README.md

```diff
@@ -63,6 +63,25 @@ aws s3 mb s3://$S3_BUCKET --region $CLUSTER_REGION
 
 Follow this [doc](https://www.kubeflow.org/docs/distributions/aws/customizing-aws/rds/#deploy-amazon-rds-mysql) to set up an AWS RDS instance. Please follow only section called `Deploy Amazon RDS MySQL`. 
 
+6. Install AWS Secrets & Configuration Provider with Kubernetes Secrets Store CSI driver
+
+Run the following commands to enable oidc and create an iamserviceaccount with permissions to retrieve secrets from AWS Secrets Manager
+
+```
+eksctl utils associate-iam-oidc-provider --region=$CLUSTER_REGION --cluster=$CLUSTER_NAME --approve
+
+kubectl create namespace kubeflow
+
+eksctl create iamserviceaccount  --name kubeflow-secrets-manager-sa  --namespace kubeflow  --cluster $CLUSTER_NAME --attach-policy-arn \ arn:aws:iam::aws:policy/AmazonSSMReadOnlyAccess --attach-policy-arn arn:aws:iam::aws:policy/SecretsManagerReadWrite --override-existing-serviceaccounts   --approve --region $CLUSTER_REGION
```

Comment:
Does this have to be namespace scoped? Is it safe/possible to reuse the same SA across all namespaces?

Reply:
the service account is in the same namespace that the ASCP and secrets mount will be and I have chosen for those to reside in the kubeflow namespace. I haven't tried making the service account outside of the namespace since this is following existing documentation. Although I don't see a reason why it wouldn't be safe to reuse the same SA across all namespaces.